test/interval_tests/numeric.jl

```diff
@@ -340,3 +340,19 @@ end
 
 
 end
+
+@testset "Mince for `Interval`s" begin
+    v = mince(-1 .. 1, 4)
+    @test v == [-1 .. -0.5, -0.5 .. 0, 0 .. 0.5, 0.5 .. 1]
+    h = hull(hull(v[1], v[2]), hull(v[3], v[4]))
```

Comment:
You can do `reduce(∪, v)`. We should make `hull(v)` for a vector `v` of intervals if this doesn't already exist.

Reply:
I was implementing it as `hull(a, b, c...) = hull( hull(a,b), c...)`, which works. I'll check it with `reduce` as well, which may be more elegant.

Reply:
Shall I use this PR to address this? 

We should also implement `hull` for `IntervalBox`es, as the hull in each direction. Does this make any sense?

Reply:
Yes please go ahead and do this in this PR. `hull` definitely makes sense for `IntervalBox`es yes.  Isn't `hull` just the same as `union`?

Reply:
In our package they are identical, since `union` is some synonym of `hull`.

Yet, I think the difference is that `union` is the set union of the intervals, while `hull(a,b)` returns an interval/intervalbox, which contains `a` and `b`.

Reply:
I'll push a commit which implements hull for intervals, and uses splatting for the hull of more than two intervals/interval-boxes.

Reply:
I believe that `reduce` is more efficient.

Reply:
I've just pushed a new commit which addresses this.